modular_skyrat/master_files/code/datums/quirks/negative.dm

```diff
@@ -21,3 +21,75 @@
 		return
 	to_chat(quirk_holder, span_warning("The nerve staple suddenly falls off your face and melts[istype(quirk_holder.loc, /turf/open/floor) ? " on the floor" : ""]!"))
 	qdel(staple)
+
+// Re-labels TG brainproblems to be more generic. There never was a tumor anyways!
+/datum/quirk/item_quirk/brainproblems
+	name = "Brain Degeneration"
+	desc = "You have a lethal condition in your brain that is slowly destroying it. Better bring some mannitol!"
+	medical_record_text = "Patient has a lethal condition in their brain that is slowly causing brain death."
+
+// Override of Brain Tumor quirk for robotic/synthetic species with posibrains.
+// Does not appear in TGUI or the character preferences window.
+/datum/quirk/item_quirk/brainproblems/synth
+	name = "Positronic Cascade Anomaly"
+	desc = "Your positronic brain is slowly corrupting itself due to a cascading anomaly. Better bring some liquid solder!"
+	gain_text = "<span class='danger'>You feel glitchy.</span>"
+	lose_text = "<span class='notice'>You no longer feel glitchy.</span>"
+	medical_record_text = "Patient has a cascading anomaly in their brain that is slowly causing brain death."
+	icon = "bp_synth_brain"
+	mail_goodies = list(/obj/item/storage/pill_bottle/liquid_solder/braintumor)
+	hidden_quirk = TRUE
+
+// If brainproblems is added to a synth, this detours to the brainproblems/synth quirk.
+// TODO: Add more brain-specific detours when PR #16105 is merged
+/datum/quirk/item_quirk/brainproblems/add_to_holder(mob/living/new_holder, quirk_transfer)
+	if(!(is_species(new_holder, /datum/species/robotic) && (src.type == /datum/quirk/item_quirk/brainproblems)))
```

Comment:
for the first one i'm fairly sure there's an `isrobotic` macro
for the second use `istype`

Reply:
Looking into this istype` function, and it is not do what I need it to do. It does not behave as you would expect in this situation. Using it causes an infinite loop and it fails to differentiate between certain objects.

Reply:
Changed `is_species` in commit b4d597862c5e1e7aee0daec7868bdf1f6d080817